layouts/page/virtual-backgrounds.html

```diff
@@ -0,0 +1,60 @@
+{{ define "hero" }}
+    <header>
+        <div class="container mx-auto p-4 text-center flex-col m-24 mb-12">
```

Comment:
```suggestion
        <div class="container mx-auto p-4 text-center m-24 mb-12 max-w-5xl">
```

... just to keep things constrained horizontally. (Otherwise it's a lot of side-to-side scanning. Rule of thumb: [keep it to less than 80 characters or so per line](https://www.smashingmagazine.com/2014/09/balancing-line-length-font-size-responsive-web-design/) for text you want people to read.)

Also, this is a lot of text for a hero section, especially on mobile. Maybe revise down a bit?

<img width="393" alt="image" src="https://user-images.githubusercontent.com/274700/107589761-8affac00-6bbb-11eb-88c7-fa10273d1443.png">

Reply:
👍